### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by DjMomo**
Add Daikin Emura FVXS hardware support.

Update set_control_info (GET http request capability on firmware 2.6.0)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ The solution provided by Daikin is a mobile app (very well designed) that allows
 
 Even if the system works, there are some problems:
 
-- It is no possible to control the AC from a web browser. You must use the official mobile application. If your device is not supported or you are from a computer you can not control your air conditioner over the internet.
+- It's possible to control the AC from a web browser. You can use the official mobile application also. If your device is not supported or you are from a computer you can not control your air conditioner over the internet.
```
Comment (ael-code) [inline README.md]: I think you misunderstood this section... This is about the official tools provided by Daikin, and as far as I know there is no official web application to control AC units.

File: README.md [hunk 2]
```diff
@@ -45,15 +52,15 @@ Uri                | GET | POST | desc
 /common/set_remote_method | | X | Set information on the polling system (reduce remote time update ??)
 /aircon/get_model_info | X | | Provides model informarion
 /aircon/get_control_info | X | | Main Uri to request all current status parameters
-/aircon/set_control_info | | X | Main Uri to set status parameters ( control almost all)
+/aircon/set_control_info | X | X | Main Uri to set status parameters (control almost all)
```
Comment (ael-code) [inline README.md]: What the AC return on the `/aircon/set_control_info` `GET` request ?

**Comment by jluke1** [comment]
In README you say.
Please contact me if you try new configurations.

This software works fine with
ModelName:          Daikin Cora FTXM35QVMA
ModelID:            FTXM35QVMA
WifiControllerID:   BRP072A42 4P384919-2C
Software version:   2.9.0

Thank you for the fine software.

File: README.md (hunk 2 above)
Comment (DjMomo) [inline README.md]: Hello,

pi@dev:~ $ curl -i "http://192.168.0.5/aircon/set_control_info?pow=1&mode=3&stemp=24.0&shum=0&f_rate=A&f_dir=0"
HTTP/1.0 200 OK
Content-Length: 11
Content-Type: text/plain

ret=OK,adv=

It works perfectly. There is an identical response if you call URL through web browser.

File: README.md (hunk 1 above)
Comment (DjMomo) [inline README.md]: Ok for this section. I understand now.

**Comment by DjMomo** [comment]
I've updated README again.

Tell me if you want more details.

**Comment by ael-code** [comment]
Thanks for the contribution 👍

**Pull request merged**